README.md

```diff
@@ -28,7 +28,7 @@ _All pages, expect the home page, require authentication._
 
 ## Dependencies
 
-We use npm and yarn to manage front-end dependencies. You should have [npm installed](https://nodejs.org/en/) as well as [yarn](https://yarnpkg.com/lang/en/docs/install/).
+We use node and yarn to manage front-end dependencies. You should have [node installed](https://nodejs.org/en/) as well as [yarn](https://yarnpkg.com/lang/en/docs/install/).
```

Comment:
Should we update `package.json` too, to remove `npm` from the scripts?

Reply:
I'll do that as part of #789

Reply:
👍